src/main/java/com/spikes2212/command/drivetrains/commands/DriveTank.java

```diff
@@ -26,14 +27,21 @@ public class DriveTank extends CommandBase {
      * @param drivetrain         the drivetrain this command requires and moves.
      * @param leftSpeedSupplier  the double {@link Supplier} supplying the speed to move on the left side with.
      * @param rightSpeedSupplier the double {@link Supplier} supplying the speed to move on the right side with.
+     * @param squareInput whether to square the speeds or not.
      */
     public DriveTank(TankDrivetrain drivetrain, Supplier<Double> leftSpeedSupplier,
-                     Supplier<Double> rightSpeedSupplier, Supplier<Boolean> isFinished) {
+                     Supplier<Double> rightSpeedSupplier, Supplier<Boolean> isFinished, boolean squareInput) {
         addRequirements(drivetrain);
         this.tankDrivetrain = drivetrain;
         this.leftSpeedSupplier = leftSpeedSupplier;
         this.rightSpeedSupplier = rightSpeedSupplier;
         this.isFinished = isFinished;
+        this.squareInput = squareInput;
+    }
+
+    public DriveTank(TankDrivetrain drivetrain, Supplier<Double> leftSpeedSupplier,
+                     Supplier<Double> rightSpeedSupplier, Supplier<Boolean> isFinished) {
+        this(drivetrain, leftSpeedSupplier, rightSpeedSupplier, isFinished, false);
     }
 
     public DriveTank(TankDrivetrain drivetrain, Supplier<Double> leftSpeedSupplier,
```

Comment:
add `squareInputs` to this (and other construrctos if needed).
turn all `boolean isFinished` to `Supplier<Boolean> isFinished` in order to prevent similar parameter types in different constrcutors.